PIM/Integration/06_ManageCatalogs.md

```diff
@@ -0,0 +1,266 @@
+
+1. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the catalog name in the left column.   
+  The menu is displayed.
+
+  ![Menu](../../Assets/Screenshots/PIM/Settings/Catalogs/Menu.png "[Menu]")
+
+2. Click the *+ Create new* menu entry.   
+  The *Create new catalog* view is displayed.
+
+  ![Create new catalog](../../Assets/Screenshots/PIM/Settings/Catalogs/CreateNewCatalog.png "[Create new catalog]")
+
+3. Enter a name for the catalog in the text field in the header.    
+
+4. Continue to [add a category](#create-a-category) to the catalog, as at least one category per catalog is required to which you can assign the products.
+
+
+
+### Create a category
+
+You need to add at least one category to a catalog to be able to assign products to it.
+
+#### Prerequisites
+
+The catalog basic data has been created, see [Define the catalog basic data](#define-the catalog-basic-data).
+
+#### Procedure
+*PIM > Settings > Tab CATALOGS > Button Create new*
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+  A plus sign is displayed in the category column.
+
+  ![Add category](../../Assets/Screenshots/PIM/Settings/Catalogs/AddCategory.png "[Add category]")
+
+2. Click the plus sign in the category column.    
+  The *Add Element* window is displayed in the category column.
+
+  ![Add element](../../Assets/Screenshots/PIM/Settings/Catalogs/AddElementTop.png "[Add element]")
+
+3. Enter a name for the category of the catalog in the *Name* field.
+
+4. Enter a key for the category of the catalog in the *Key* field.
+
+5. Click the [SAVE] button.   
+  The catalog category has been created. The *Add Element* window is closed. The new category is displayed in the category column.
+
+  ![Category created](../../Assets/Screenshots/PIM/Settings/Catalogs/CategoryCreated.png "[Category created]")
+
+  > [Info] To add an additional category, repeat the steps **1** to **5**. To add a sub-category, see [Create a sub-category](#create-a-sub-category).  
+
+6. Click the [SAVE] button in the upper right corner.  
+  The new catalog has been saved. The *Create new catalog* view is closed.
+
+
+
+## Edit a catalog
+
+After you have created a catalog, you can edit the catalog and its categories. You can create further categories, change the order of categories and delete categories.
+
+### Create a sub-category
+
+Create a sub-category to be able to assign products to individual categories in an even more detailed way.
+
+#### Prerequisites
+
+At least one catalog with one category has been created, see [Create a catalog](#create-a-catalog).
+
+#### Procedure
+*PIM > Settings > Tab CATALOGS*
+
+![Catalogs](../../Assets/Screenshots/PIM/Settings/Catalogs/Catalogs.png "[Catalogs]")
+
+1. Select the catalog you want to create a sub-category to in the drop-down list in the left column.   
```

Comment:
```suggestion
1. Select the catalog where you want to create a sub-category in the drop-down list in the left column.   
```
Unsicher über den Ausdruck "create a category to..." (=in welchem Katalog). Where?